packages/core/src/external/aws/lambda-logic/document-uploader.ts

```
@@ -77,16 +80,18 @@ export async function documentUploaderHandler(
       const message = "Failed with the call to update the doc-ref of an uploaded file";
       console.log(`${message}: ${docRef}`);
```

Do we really want to fail silently here?